proposals/0003-pr-process.md

```diff
@@ -18,6 +20,8 @@ PRs fall into two categories:
 
     Uncontroversial changes can be approved by either one committee member who did not work on the PR or two members who worked on a PR together.
 
+    Committee members should exercise good judgement on what constitutes working together for the sake of this process. Pairing on a call or tightly collaborating asynchronously probably count. A committee member having contributed to a feature branch several weeks before it is ready to be merged probably doesn't count.
+
```

Comment:
As discussed in the call today I'll submit a PR clarify approval of the uncontroversial changes.

Reply:
I think that after https://github.com/haskell-org/committee/pull/7/files this new paragraph is actually redundant. We don't phrase the policy in terms of "working together" any more.